libsession/src/main/java/org/session/libsession/utilities/Util.kt

```diff
@@ -383,3 +383,9 @@ fun <T, K: Any, V: Any> Iterable<T>.associateByNotNull(
         this[key] = value
     }
 }
+
+inline fun <T, K> Iterable<T>.groupByNotNull(keySelector: (T) -> K?): Map<K, List<T>> {
```

Comment:
Would be nice to add a comment about what this function does - it's pretty hard to parse in your head

Reply:
done